lib/client-assets.php

```diff
@@ -1093,6 +1092,7 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	// https://github.com/WordPress/gutenberg/issues/5667.
 	add_filter( 'user_can_richedit', '__return_true' );
 
+	wp_enqueue_script( 'wp-utils' );
```

Comment:
We should have commented to explain the context for this line. I'm in the course of removing the deprecated `./utils` and it wasn't clear to me what relation this has.